### PR TITLE
hw-9

### PR DESCRIPTION
**Pull request by GigiDev7**

### CODE REVIEW

File: client/src/app/auth/services/auth.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    service = TestBed.inject(AuthService);
+    httpController = TestBed.inject(HttpTestingController);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should return user data on success', (done: DoneFn) => {
+    const expectedeData = {
```
Comment (AnastasiaHarost) [inline client/src/app/auth/services/auth.service.spec.ts]: move this constant to the top of the file, then you can reuse it in 2 places

**Review by AnastasiaHarost — APPROVED** [review]

**Comment by GigiDev7** [comment]
Last homework, it was good working with you. Good luck

**Comment by AnastasiaHarost** [comment]
> Last homework, it was good working with you. Good luck

Thanks for your hard-working and speed in completing homeworks. Good luck on the interview, hope to meet you at Exadel:)

**Comment by GigiDev7** [comment]
> > Last homework, it was good working with you. Good luck
> 
> Thanks for your hard-working and speed in completing homeworks. Good luck on the interview, hope to meet you at Exadel:)

Thank you very much

**Pull request merged**